stacks-signer/src/main.rs

```diff
@@ -52,7 +52,7 @@ use stacks_signer::runloop::{RunLoop, RunLoopCommand};
 use stacks_signer::utils::{build_signer_config_tomls, build_stackerdb_contract};
 use tracing_subscriber::prelude::*;
 use tracing_subscriber::{fmt, EnvFilter};
-use wsts::state_machine::coordinator::frost::Coordinator as FrostCoordinator;
+use wsts::state_machine::coordinator::fire::Coordinator as FrostCoordinator;
```

Comment:
NIT: Should probably call this FireCoordinator instead of FrostCoordinator

Reply:
Yeah, I was trying to keep the changes minimal, but it's def confusing this way.

I think I'll call it CoordinatorStateMachine or something else neutral, so we can just import whatever with that name.

Reply:
I went ahead and called it `FireCoordinator`